tests/migrate_test.go

```diff
@@ -54,6 +54,27 @@ func TestMigrate(t *testing.T) {
 	}
 }
 
+func TestAutoMigrateSelfReferential(t *testing.T) {
+	type MigratePerson struct {
+		ID        uint
+		Name      string
+		ManagerID *uint
+		Manager   *MigratePerson
+	}
+
+	if err := DB.Debug().Migrator().DropTable(&MigratePerson{}); err != nil {
```

Comment:
maybe the drop table error check is not necessary? it will broken when running the test first time? also please remove the `Debug()` method here.

Reply:
make sense, I have removed that.